docs/installation/server-config/security.rst

```diff
@@ -36,11 +36,18 @@ Use the following settings to configure the KSQL server to require authenticatio
 ::
 
     authentication.method=BASIC
-    authentication.roles=admin,developer,user,ksq-user
-    authentication.realm=KsqlServer-Props
+    authentication.roles=<user-role1>, <user-role2>, ...
```

Comment:
Does the config work with spaces between different roles? I've never tried it like that.

Reply:
Dunno, but better safe than sorry. ;)